pkg/monitor/monitor/monitor_manager.go

```diff
@@ -806,3 +811,16 @@ func buildEtcdValue(host string, port int) (string, error) {
 	}
 	return string(data), nil
 }
+
+func (m *MonitorManager) syncBasicAuth(monitor *v1alpha1.TidbMonitor, store *Store) error {
+	if monitor.Spec.Prometheus.RemoteWrite != nil {
```

Comment:
Can we remove this `if` and use the following `for` directly?

Reply:
ok